modules/logs/logs_plugin.rb

```diff
@@ -0,0 +1,14 @@
+class ::Proxy::LogsPlugin < ::Proxy::Plugin
+  http_rackup_path File.expand_path("http_config.ru", File.expand_path("../", __FILE__))
+  https_rackup_path File.expand_path("http_config.ru", File.expand_path("../", __FILE__))
+
+  plugin :logs, ::Proxy::VERSION
+  default_settings :enabled => true
+
+  after_activation do
+    begin
+      buffer = Proxy::LogBuffer::Buffer.instance
+      logger.info "Log buffer plugin activated with #{buffer.size}/#{buffer.size_tail} capacity"
```

Comment:
Will `Log buffer API initialized, available capacity: #{buffer.size}/#{buffer.size_tail}` be better? Or do you prefer not to print anything?